src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx

```diff
@@ -70,6 +71,7 @@ const VirtualMachineNavPageTitle: FC<VirtualMachineNavPageTitleProps> = ({ name,
               <SidebarEditorSwitch />
             </SplitItem>
           )}
+          {!isEmpty(vm) && <VMActionsIconBar vm={vm} />}
```

Comment:
u can move it 2 lines below and add <> </> and remove the double check

Reply:
Done!